core/src/main/java/com/epam/pipeline/entity/pipeline/run/EngineType.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2025 EPAM Systems, Inc. (https://www.epam.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.epam.pipeline.entity.pipeline.run;
+
+public enum EngineType {
+    NEXTFLOW,
+    CWL,
```

Comment:
since we don't have a case for `CWL` and `WDL` now, I don't think we should place it here for now, let it be just nextflow